index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - setup and mobile first</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;600&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="toolbar">
+            <div class="container-1">
+                <div class="clock"><p><strong>9:41</strong></p></div>
+                <div class="space2"></div>
+                <div class="conn"><img class="conn-im" alt="Connetion" src="images/connection.png"></div>
+                <div class="wire"><img class="wire-im" alt="Wireless" src="images/wireless.png"></div>
+                <div class="batt"><img class="batt-im" alt="Battery" src="images/battery.png"></div>
+            </div>
+            <div class="container-2">
+                <div class="logo"><img class="mylogo-im" alt="MyLogo" src="images/mylogo.png"></div>
```

Comment:
- [ ] Kindly make sure  The logo has a link (`a` tag).

- This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.